### PR TITLE
docs: Add instructions to update & verify scripts

### PR DESCRIPTION
**Pull request by indradhanush**

### CODE REVIEW

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -19,6 +19,13 @@
 
 # First of all, the code-generator needs to be fetched:
 # $ go get -u k8s.io/code-generator
+#
+# Now checkout the appropriate branch for the Kubernetes version that
+# you're building for. For example if you're building for Kubernetes
+# 1.10.0 you would do the following:
```
Comment (asymmetric) [inline hack/update-codegen.sh]: Can you shorten this to "you would checkout `release-1.10`"? I don't think we need to explain everything ;)

Also, the tag is wrong.

Comment (asymmetric) [reply hack/update-codegen.sh]: Actually you know what, I was wrong on my second point. It's better to checkout the tag as you did, than the branch, as I did.

Comment (indradhanush) [reply hack/update-codegen.sh]: Fixed.

Comment (indradhanush) [reply hack/update-codegen.sh]: Ah didn't see the second comment. Fixing.

Comment (indradhanush) [reply hack/update-codegen.sh]: Fixed now.

File: hack/update-codegen.sh [hunk 2]
```diff
@@ -19,6 +19,11 @@
 
 # First of all, the code-generator needs to be fetched:
 # $ go get -u k8s.io/code-generator
+#
+# Now checkout the appropriate branch for the Kubernetes version that
+# you're building for. For example if you're building for Kubernetes
+# 1.10.0 you would do checkout the kubernetes-1.10.0 branch.
```
Comment (asymmetric) [inline hack/update-codegen.sh]: Extra "do", and please wrap `kubernetes-1.10.0` in backticks :+1:

Comment (indradhanush) [reply hack/update-codegen.sh]: Fixed the extra "do".

Regarding the backticks, this is a bash comment? Sounds odd to me to add backticks the same way we don't do that in Golang comments. Is there a conventional practice I don't know about?

Comment (asymmetric) [reply hack/update-codegen.sh]: I do that in Go comments as well ;)

Not sure if there's an official agreement but I think I've seen others do it as well.

File: hack/update-codegen.sh [hunk 3]
```diff
@@ -19,6 +19,11 @@
 
 # First of all, the code-generator needs to be fetched:
 # $ go get -u k8s.io/code-generator
+#
+# Now checkout the appropriate branch for the Kubernetes version that
+# you're building for. For example if you're building for Kubernetes
+# 1.10.0 you would checkout the `kubernetes-1.10.0 branch`.
```
Comment (elliott-davis) [inline hack/update-codegen.sh]: based on using this just now to fix my local setup, the branch name was `release-x.x.x`

Comment (indradhanush) [reply hack/update-codegen.sh]: Thanks @elliott-davis. I've pushed a fix.
